src/Components/CalculatorResults.jsx

```diff
@@ -1,87 +1,43 @@
 import React, { useEffect, useState } from "react";
-import NonElligible from "./NonElligible";
+import ResultNo from "./ResultNo";
 import ResultYes from "./ResultYes";
 
 // data import
-import calculatorNonElligible from "../data/calculatorNonElligible.json";
-//*!this page will carry the logic for determinig which result page to render --> will conditionally render the correct page based off user input
+import calculatorNonEligible from "../data/calculatorNonEligible.json";
 
-const CalculatorResults = ({ userResponse, setUserResponse }) => {
+/** This component determines which result to render based off user input */
+const CalculatorResults = ({ userResponse }) => {
   const [result, setResult] = useState(null);
-  const [nonElligible, setNonElligible] = useState([]);
+  const [nonEligible, setNonEligible] = useState([]);
+
   useEffect(() => {
     userResponse && handleCalculation();
   }, []);
 
   const handleCalculation = () => {
-    let nonElligibleItems = [];
+    let nonEligibleItems = [];
     for (const id in userResponse) {
       let value = userResponse[id];
       if (value === false) {
-        const { text } = calculatorNonElligible.filter((q) => q.id == id)[0];
-        nonElligibleItems.push(text);
+        const { text } = calculatorNonEligible.filter((q) => q.id == id)[0];
```

Comment:
That [0] is to grab the first element in the array returned by filter. I removed the issue where someone can click to get to '/calculator/0'. 

The bigger issue is that our current setup doesn't have any edge case handling. If a user types in '/calculator/one', '/calculator/banana', or any digit outside of the id's in our question array then they're going to see a broken screen. That needs to be fixed before we release, but shouldn't block this PR.

Reply:
hmmm. Good point. Is there a way react router can determine if its a number or not? Otherwise, maybe add 'exact' to the route component and then have a HOC or sub route component to handle the edge cases and deliver a "sorry page not found. Please return to ______" component.

Reply:
We'd have to write the logic to determine if it's a number or a valid route. I think it can be as simple as a reroute inside of the CalculatorQuestionsContainer.